doc/api/deprecations.md

```diff
@@ -370,14 +370,14 @@ changes:
   - version: v8.0.0
     pr-url: https://github.com/nodejs/node/pull/9683
     description: End-of-Life.
-  - version: v6.0.0
-    pr-url: https://github.com/nodejs/node/pull/4525
-    description: Runtime deprecation.
   - version:
-    - v4.8.6
     - v6.12.0
+    - v4.8.6
     pr-url: https://github.com/nodejs/node/pull/10116
     description: A deprecation code has been assigned.
+  - version: v6.0.0
```

Comment:
I think it makes sense to order them based on the highest number, but what about the next one that has a 4.x

Reply:
I think it makes sense to have them ordered in the order of "appearance" in master, and that's also the convention the majority of the docs follows.